### PR TITLE
TEN-1037 / 24.10 / Adding read only admin services test cases

### PR DESCRIPTION
**Pull request by ericbsd**

### CODE REVIEW

**Comment by ericbsd** [comment]
Screenshot from 2024-06-07 14-47-39

File: keywords/webui/common.py [hunk 1]
```diff
@@ -993,7 +998,7 @@ def is_file_downloaded(cls, download_path: str, filename: str) -> bool:
         return file.is_file()
 
     @classmethod
-    def is_input_visible(cls, name: str) -> bool:
+    def is_input_visible(cls, name) -> bool:
```
Comment (markbainix) [inline keywords/webui/common.py:1001]: missing type cast

File: keywords/webui/common.py [hunk 2]
```diff
@@ -1006,7 +1011,7 @@ def is_input_visible(cls, name: str) -> bool:
         return cls.is_visible(xpaths.common_xpaths.input_field(cls.convert_to_tag_format(name)))
 
     @classmethod
-    def is_link_visible(cls, name: str) -> bool:
+    def is_link_visible(cls, name) -> bool:
```
Comment (markbainix) [inline keywords/webui/common.py:1014]: missing type cast

**Review by markbainix — CHANGES_REQUESTED** [review]

File: keywords/webui/common.py (hunk 2 above)
Comment (ericbsd) [reply keywords/webui/common.py:1014]: Oh that must be from my merge.

**Review by markbainix — APPROVED** [review]

**Pull request merged**